src/utils/utils.js

```diff
@@ -7,3 +17,21 @@ export const getRandomElement = (array) => {
   const randomIndex = Math.floor(Math.random() * (array.length));
```

Comment:
Давай что-бы не плодить рандом логику, будет внутри переиспользовать теперь существующий `getRandomNumber`?

```suggestion
  const randomIndex = getRandomNumber(0, array.length - 1)
```